packages/bpk-component-floating-notification/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bpk-component-floating-notification",
+  "version": "1.0.0",
+  "description": "Backpack floating-notification component.",
+  "license": "Apache-2.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "bpk-mixins": "^31.1.1",
+    "bpk-react-utils": "^4.1.1",
+    "prop-types": "^15.5.8"
+  },
+  "peerDependencies": {
+    "react": "^16.3.0 || ^17.0.0"
+  },
+  "devDependencies": {
+    "bpk-component-button": "^6.2.4",
+    "bpk-component-icon": "^9.2.10",
+    "bpk-component-text": "^7.0.3"
```

Comment:
These is used in the `BpkFloatingNotification.js` file so is a direct dependency rather than a dev dep